src/common/styles/common.scss

```diff
@@ -22,14 +22,6 @@ button::after {
     position: absolute;
 }
 
-.ms-Button.ms-Button--primary:not(.is-disabled) {
-    background: $communication-primary;
-
-    &:hover {
-        background: $communication-shade-10;
```

Comment:
Dropping this hover style is intentional; it results in using a reasonable looking white hover color that Fer was happy with